app/views/voice/otp.xml.slim

```diff
@@ -0,0 +1,7 @@
+doctype xml
```

Comment:
Yeah I'm open to `.xml.erb`-ing this too? I figured since it's a dealbreaker to have malformed XML for the Twilio API, I'd make it harder to mess up? But I find ERBs easier to read sometimes